### PR TITLE
Features branch

### PR DESCRIPTION
**Pull request by nurislam-ab**
**Completed a clone of old version of the Apple website.**

The clone was built using display property and background images and gradients. The clone consists of:
- navigation bar
- hero section
- product/services section
- footer

### CODE REVIEW

File: main.css [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+    color:#08c
+}
+
+.hero-text a{
+    text-decoration: none;
+    color:white;
+}
+
+.menu{
+    display: grid;
```
Comment (pelzolga123) [inline main.css]: Use float OR flex instead.

File: main.css [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+    position: relative;
+    display: block;
+}
+
+.container h2{
+    display: none;
+}
+
+.content{
+    display: grid;
```
Comment (pelzolga123) [inline main.css]: Use float OR flex instead.

File: main.css [hunk 3]
```diff
@@ -0,0 +1,296 @@
...
+    width:100%;
+}
+
+footer {
+    margin:auto;
+    width:100%;
+}
+
+footer .container{
+    display: grid;
```
Comment (pelzolga123) [inline main.css]: Use float OR flex instead.

**Review by pelzolga123 — CHANGES_REQUESTED** [review]
Project requires alignment changes.

**Comment by pelzolga123** [comment]
## Hi Nurislam!  
### Your project looks :+1: amazing!
Just a few things to correct:
* [x] Pull Requests should have a short summary of changes introduced here. Right now it is too general, [link to Pathwright instruction](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/).
* [x] Elements should be aligned with float OR flex. [Link to screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/building_with_backgrounds_and_gradients/float_or_flex_requirement.png).

### Please kindly Fix the above issue and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

**Comment by shshamim63** [comment]
## Status :green_apple:  [Approved] :green_apple: 
___
Hi @nurislam-ab :wave: ,
Awesome :clap: . You have followed all the task that was mentioned by previous reviewer. Code looks neat and clean. No error on HTML markup. So the project is complete.
___
#####  This :+1: is from me. You can merge it. And don't forget to fill the project completion form.
___
## :herb:  Happy Coding :herb: 
### _Shakhawat Hossain_
[Linkedin](https://www.linkedin.com/in/shakhawathossainshamim/) | [Slack](Shakhawat Hossain)

**Pull request merged**